README/readme.md

```diff
@@ -44,68 +48,70 @@ The basic Open ID Connect set is:
-this is not the final set of services, there will be more. And there will be variants of the standard ones. Like when you
-want to add multi lateral federation support to provider information discovery and client registration.
+To these one can add services like session management and token introspection. The
+only thing we can be sure of is that this is not the final set of services, there
+will be more. And there will be variants of the standard ones. Like when you want 
+to add multi lateral federation support to provider information discovery and
+client registration.
 
-Over all it seemed like a good idea to write a piece of code that implements all the functionality that is needed to support
-any of this services and any future services that follows the same pattern.
+Over all it seemed like a good idea to write a piece of code that implements all 
+the functionality that is needed to support any of this services and any future 
+services that follows the same pattern.
 
 
 ### The request pipeline
-Below follows a desciption of the parts of the request pipeline in the order they are called.
+Below follows a description of the parts of the request pipeline in the order they
+are called.
 
 The overall call sequence looks like this:
-
     do_request_init
-
       request_info
-  
         construct
-    
           do_pre_construct (#)
-      
           gather_request_args
-      
           do_post_construct (#)
-      
       init_authentication_method
-  
       uri_and_body
-  
         endpoint
-    
       update_http_args
   
   
-The result of the request pipeline is a dictionary that in its simplest form will look something like this:
+The result of the request pipeline is a dictionary that in its simplest form will
+look something like this:
+
 ```
 {
-    'uri' : 'https://example.com/authorize?response_type=code&state=state&client_id=client_id&scope=openid&redirect_uri
-            =https%3A%2F%2Fexample.com%2Fcli%2Fauthz_cb&nonce=P1B1nPCnzU4Mwg1hjzxkrA3DmnMQKPWl'
+  'uri'
+      : 'https://example.com/authorize?response_type=code&state=stateclient_id=client_id&scope=openid&redirect_uri = https % 3A % 2F %2Fexample.com % 2Fcli % 2Fauthz_cb&nonce = P1B1nPCnzU4Mwg1hjzxkrA3DmnMQKPWl'
 }
 ```
 
-It will look like that when the request is to be transmitted as the urlencoded query part of a HTTP GET operation. If 
-instead a HTTP POST with a json body is expected the outcome of do_request_init will be something like this:
-
-```
-{
-    'uri': 'https://example.com/token',
-    'body': 'grant_type=authorization_code&redirect_uri=https%3A%2F%2Fexample.com%2Fcli%2Fauthz_cb&code=access_code&
-            client_id=client_id',
-    'h_args': {'headers': {'Authorization': 'Basic Y2xpZW50X2lkOnBhc3N3b3Jk', 'Content-Type': 'application/x-www-form-url
-    encoded'}}
-}
+    It will look like that when the request is to be transmitted
```

Comment:
I think you didn't mean to add these indentations (See https://github.com/GJWT/nodeOIDCCli/blob/c425742916b631d95a43d47f68a69466cf3294f1/README/readme.md#the-request-pipeline how it is formatted)